metapool/sample_sheet.py

```diff
@@ -257,6 +257,15 @@ def _is_empty(csv_line):
                     section[key] = value
                     continue
 
+    def set_override_cycles(self, value):
+        # assume that any value including None is valid.
+        # None should be silently converted to empty string as a truly
+        # empty value would be better than None or na.
+        if value is None:
+            value = ''
+
+        self.Settings['OverrideCycles'] = value
```

Comment:
Likewise, the string 'OverrideCycles' is already used 5 times at various places in sample_sheet (this will make six :) so I'd really love to see it replaced (throughout) with a constant.